01-syntax/03-data-structures/the-sum-of-ranges/range.js

```diff
@@ -9,3 +9,7 @@
 console.log(range(5, 2, -1));
 
 // → [5, 4, 3, 2]
+
+console.log(range(10, 1));
```

Comment:
More test cases... 
For example
What about calling range with only one parameter ?